tools/python/boutiques/schema/examples/baremetal/test_baremetal.json

```diff
@@ -0,0 +1,30 @@
+{
+    "command-line": "echo \"Bare metal execution\" > [FILENAME] && cat [FILENAME]", 
+    "container-image": {
+        "entrypoint": false, 
+        "image": "neurodata/ndmg:v0.0.41-1", 
+        "index": "index.docker.io", 
+        "type": "docker"
+    },
```

Comment:
Could you confirm that this still works if `container-image` isn't present in the descriptor?

Reply:
I tried the following descriptor/flag combination with `bosh exec launch` and they all behaved as expected:
no container-image -> exec in command line
no container-image + --no-container -> exec in command line
w/ container-image -> exec in container
w/ container-image + --no-container -> exec in command line